src/index.js

```diff
@@ -3,5 +3,14 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import 'bootstrap/dist/css/bootstrap.css';
+import {
+    BrowserRouter as Router,
+    Route
```

Comment:
you only need `BrowserRouter as Router` here since you only call `Route` within App.js